src/PowerShellEditorServices/Services/TextDocument/Handlers/CodeActionHandler.cs

```diff
@@ -52,13 +53,13 @@ public override async Task<CommandOrCodeActionContainer> Handle(CodeActionParams
         {
             if (cancellationToken.IsCancellationRequested)
             {
-                _logger.LogDebug("CodeAction request canceled at range: {0}", request.Range);
+                _logger.LogDebug($"CodeAction request canceled at range: {request.Range}");
                 return Array.Empty<CommandOrCodeAction>();
             }
 
-            // On Windows, VSCode still gives us file URIs like "file:///c%3a/...", so we need to escape them
             IReadOnlyDictionary<string, MarkerCorrection> corrections = await _analysisService.GetMostRecentCodeActionsForFileAsync(
```

Comment:
Can we change this to:
```
IReadOnlyDictionary<string, MarkerCorrection> corrections = null;
            try
            {
                var file = _workspaceService.GetFile(request.TextDocument.Uri);
                corrections = await _analysisService.GetMostRecentCodeActionsForFileAsync(file).ConfigureAwait(false);
            }
            catch (FileNotFoundException ex)
            {
                _logger.LogWarning("textDocument/codeAction FileNotFound {0}", request.TextDocument.Uri);
                return Array.Empty<CommandOrCodeAction>();
            }
```

Reply:
That change was made to `GetMostRecentCodeActionsForFileAsync` itself in fcf2ee7d8221a1ffe883ed54ea4542228515dd9b

Reply:
Ah perfect, thanks!